tests/pr_test.go

```diff
@@ -108,3 +118,85 @@ func TestRunUpgradeSolutionInstances(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestAgentsSolutionInSchematics(t *testing.T) {
+	t.Parallel()
+
+	var region = validRegions[rand.Intn(len(validRegions))]
+
+	// ------------------------------------------------------------------------------------------------------
+	// Deploy SLZ ROKS Cluster and Observability instances since it is needed to deploy Observability Agents
+	// ------------------------------------------------------------------------------------------------------
+
+	prefix := fmt.Sprintf("slz-%s", strings.ToLower(random.UniqueId()))
+	realTerraformDir := "./resources"
+	tempTerraformDir, _ := files.CopyTerraformFolderToTemp(realTerraformDir, fmt.Sprintf(prefix+"-%s", strings.ToLower(random.UniqueId())))
+
+	// Verify ibmcloud_api_key variable is set
+	checkVariable := "TF_VAR_ibmcloud_api_key"
+	val, present := os.LookupEnv(checkVariable)
+	require.True(t, present, checkVariable+" environment variable not set")
+	require.NotEqual(t, "", val, checkVariable+" environment variable is empty")
+
+	logger.Log(t, "Tempdir: ", tempTerraformDir)
+	existingTerraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: tempTerraformDir,
+		Vars: map[string]interface{}{
+			"prefix": prefix,
+		},
+		// Set Upgrade to true to ensure latest version of providers and modules are used by terratest.
+		// This is the same as setting the -upgrade=true flag with terraform.
+		Upgrade: true,
+	})
+
+	terraform.WorkspaceSelectOrNew(t, existingTerraformOptions, prefix)
+	_, existErr := terraform.InitAndApplyE(t, existingTerraformOptions)
+
+	if existErr != nil {
+		assert.True(t, existErr == nil, "Init and Apply of temp resources (SLZ-ROKS and Observability Instances) failed")
+	} else {
+
+		rbacSynchSleepTime := 180
+		logger.Log(t, fmt.Sprintf("Sleeping for %d seconds to allow RBAC to sync", rbacSynchSleepTime))
+		time.Sleep(time.Duration(rbacSynchSleepTime) * time.Second)
```

Comment:
RBAC sync only occurs after you have accessed the cluster for the first time (so in terraform that occurs after a data lookup happens for the first time). So adding a sleep here is actually doing nothing I believe. Did you run into issues with RBAC sync here?

Reply:
This one...I referred from liberty operator repo. I was not sure if its needed. Currently ran test without this and it passed. So will remove it.

Reply:
Removed.